src/field_theory/ratfunc.lean

```diff
@@ -137,18 +137,18 @@ of `∀ {p q a : K[X]} (hq : q ≠ 0) (ha : a ≠ 0), f (a * p) (a * q) = f p q)
 -/
 @[irreducible] protected def lift_on {P : Sort v} (x : ratfunc K)
   (f : ∀ (p q : K[X]), P)
-  (H : ∀ {p q p' q'} (hq : q ∈ K[X]⁰) (hq' : q' ∈ K[X]⁰), p * q' = p' * q →
+  (H : ∀ {p q p' q'} (hq : q ∈ K[X]⁰) (hq' : q' ∈ K[X]⁰), q' * p = q * p' →
```

Comment:
@jjaassoonn, do you remember why you changed this in dfcc1206c4b51f6f7ae7b1cf146ca53282b61328?

Reply:
I just tested the original version. Without flipping p and q, the `H q.2 q’.2` on line 143 would not compile.